backend/czi_hosted/compute/diffexp_cxg.py

```diff
@@ -35,7 +35,7 @@ def get_thread_executor():
     return diffexp_thread_executor
 
 
-def diffexp_ttest(adaptor, maskA, maskB, top_n=8, diffexp_lfc_cutoff=0.01):
+def diffexp_ttest(adaptor, maskA, maskB, top_n=8, diffexp_lfc_cutoff=0.01, two_lists=False):
```

Comment:
why retain the old "single list" mode?  Is it in use?  If not, it seems that code and test simplicity should be preferable to adding another modality....